### DISCUSSION TITLE
Update node-zwave-js in running IOBroker Instance

### DISCUSSION THREAD

**Discussion by LaplaceII** [zwave-js/zwave-js#7120] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

Hello, is there any possibility to update the Node-zwave-js Database (see picture) to 13.2.0 without reinstalling the iobroker zwave2 adapter? Not sure if new installation will help....what I have seen on iobroker zwave2 github is that 10.23.6 the latest supported one. Any chance to update it to 13.2.0?

![Unbenannt](https://github.com/user-attachments/assets/d7450ce2-7544-4517-aed6-a7b5ca3ef68e)

### Software versions

IOBroker: ZWave2 v3.1.0, node-zwave-js 10.23.6, JS Controller 5.0.19
Node.JS: 20.15
NPM: 10.70

### Device information

Manufacturer: ...
Model name: ...
Node ID: ...

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment]
Unfortunately there were quite a few breaking changes, which require changes to the adapter. 
I started working on those a while ago, but haven't finished yet.

**Reply by LaplaceII** [reply]
Thanks! Understood! Looking forward...
